pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -48,8 +48,11 @@ const (
 	DeployerRoleName          = "system:deployer"
 	RouterRoleName            = "system:router"
 	RegistryRoleName          = "system:registry"
+	NodeRoleName              = "system:node"
+	SDNReaderRoleName         = "system:sdn-reader"
+	SDNManagerRoleName        = "system:sdn-manager"
 	InternalComponentRoleName = "system:component"
-	DeleteTokensRoleName      = "system:delete-tokens"
+	DeleteOAuthTokensRoleName = "system:delete-oauth-tokens"
```

Comment:
"system:oauth-token-deleter" so that the binding can just be a pluralized version?